pallets/did/src/tests.rs

```diff
@@ -41,7 +43,7 @@ fn validate_delegated_claim() {
         System::set_block_number(1);
 
         // Predefined delegate type: "Sr25519VerificationKey2018"
-        let delegate_type = b"x25519VerificationKey2018".to_vec();
```

Comment:
Still don't know what this is?

Reply:
Afraid not, but I made the value configurable at least now.